src/resources/views/crud/fields/date_range.blade.php

```diff
@@ -47,8 +47,8 @@ function formatDate($entry, $dateFieldName)
 ?>
 
 @include('crud::fields.inc.wrapper_start')
-    <input class="datepicker-range-start" type="hidden" name="{{ $field['name'][0] }}" value="{{ old(square_brackets_to_dots($field['name'][0])) ?? $start_value ?? $start_default ?? '' }}">
-    <input class="datepicker-range-end" type="hidden" name="{{ $field['name'][1] }}" value="{{ old(square_brackets_to_dots($field['name'][1])) ?? $end_value ?? $end_default ?? '' }}">
+    <input class="datepicker-range-start" type="hidden" name="{{ $field['name'][0] }}" value="{{ oldValueDefaultOrFallback($field, '', $field['name'][0], $start_default, $start_value ?? null) }}">
+    <input class="datepicker-range-end" type="hidden" name="{{ $field['name'][1] }}" value="{{ oldValueDefaultOrFallback($field, '', $field['name'][1], $end_default, $end_value ?? null) }}">
```

Comment:
This is another proof that `oldValueDefaultOrFallback()` should not accept a field. If it only returned an input value, all this mumbo-jumo wouldn't be needed.

Reply:
I think this is the proof that we should not have fields (i mean field!!) that has array name defined in crud panel. 

It's the only field that behaves this way, and it's not even because is necessary, a simple config `'start_column => 'start''` would avoid code that reached crud panel class to deal with arrayed names!

I should have spotted this bottleneck sooner! :|